### DISCUSSION TITLE
`duckdb_arrow_array_scan` not present in 0.8.1 but is listed in the C API?

### DISCUSSION THREAD

**Discussion by jeremyosterhoudt** [duckdb/duckdb#8463] [Q&A] [answered]
Hello,

I'm working on interfacing with DuckDB and Apache Arrow using the C API.  There is a method we wanted to use called `duckdb_arrow_array_scan` listed in the [C API documentation](https://duckdb.org/docs/api/c/api.html#duckdb_arrow_array_scan) but is not present in the [0.8.1 tagged version of the `duckdb.h`](https://github.com/duckdb/duckdb/blob/v0.8.1/src/include/duckdb.h).

Couple of questions around this:

- Is this function meant to be exposed?
- If so, is there a plan to do another release in the near future?

**Comment by Tishj** [comment]
What you linked is the dev version, not 0.8.1
This is 0.8.1:
https://duckdb.org/docs/archive/0.8.1/api/c/api

There are already ideas to make this more explicit, but it already is visible, notice the "dev" in the top right.

**Reply by jeremyosterhoudt** [reply]
That's my bad, I missed that.  I went to the main site for which I assumed it would be based on the latest release documentation and missed the switch in the upper right. 

Are there any plans for a release in the near future or best to build a dev version to use that API for now?

**Reply by carlopi** [reply, marked as answer]
https://duckdb.org/dev/release-dates

**Reply by jeremyosterhoudt** [reply]
Thank so much!!
